Index: src/sample/search.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/sample/search.fxml b/src/sample/search.fxml
--- a/src/sample/search.fxml	(revision 8d1c89b7510e1a7c8ba54b9b078d748d9a9e322f)
+++ b/src/sample/search.fxml	(date 1616168276000)
@@ -53,7 +53,7 @@
          <children>
             <DatePicker fx:id="dateFrom" layoutX="14.0" layoutY="90.0" prefHeight="27.0" prefWidth="120.0" promptText="from" />
             <DatePicker fx:id="dateTo" layoutX="154.0" layoutY="90.0" prefHeight="27.0" prefWidth="120.0" promptText="to" />
-            <Button fx:id="buttonFindTour" layoutX="290.0" layoutY="90.0" mnemonicParsing="false" onAction="#buttonFindTourOnAvtion" prefHeight="27.0" prefWidth="127.0" text="Find tour" />
+            <Button fx:id="buttonFindTour" layoutX="290.0" layoutY="90.0" mnemonicParsing="false" prefHeight="27.0" prefWidth="127.0" text="Find tour" />
             <Text layoutX="14.0" layoutY="54.0" strokeType="OUTSIDE" strokeWidth="0.0" text=" ICELAND" wrappingWidth="164.00634765625">
                <font>
                   <Font size="36.0" />
@@ -65,11 +65,11 @@
                </children>
             </AnchorPane>
             <Text layoutX="108.0" layoutY="74.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Day tours" />
-            <Button fx:id="buttonSearchBookTour" layoutX="274.0" layoutY="306.0" mnemonicParsing="false" onAction="#buttonSearchBookTourOnAction" prefHeight="27.0" prefWidth="144.0" text="Next step to book tour" />
-            <Button fx:id="buttonAdministrator" layoutX="329.0" layoutY="14.0" mnemonicParsing="false" onAction="#buttonAdministratorOnAction" text="Administrator" />
+            <Button fx:id="buttonSearchBookTour" layoutX="274.0" layoutY="306.0" mnemonicParsing="false" prefHeight="27.0" prefWidth="144.0" text="Next step to book tour" />
+            <Button fx:id="buttonAdministrator" layoutX="329.0" layoutY="14.0" mnemonicParsing="false" text="Administrator" />
             <Text layoutX="17.0" layoutY="324.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Cancel your booking:" />
             <TextField fx:id="cancelBookingID" layoutX="16.0" layoutY="334.0" prefHeight="27.0" prefWidth="135.0" promptText="Enter your booking ID" />
-            <Button fx:id="buttonCancel" layoutX="160.0" layoutY="334.0" mnemonicParsing="false" onAction="#buttonCancelOnAction" prefHeight="27.0" prefWidth="61.0" text="Cancel" />
+            <Button fx:id="buttonCancel" layoutX="160.0" layoutY="334.0" mnemonicParsing="false" prefHeight="27.0" prefWidth="61.0" text="Cancel" />
          </children>
       </AnchorPane>
    </center>
Index: .idea/workspace.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/workspace.xml b/.idea/workspace.xml
--- a/.idea/workspace.xml	(revision 8d1c89b7510e1a7c8ba54b9b078d748d9a9e322f)
+++ b/.idea/workspace.xml	(date 1616168350000)
@@ -3,6 +3,12 @@
   <component name="ChangeListManager">
     <list default="true" id="cdb932d9-7e58-4d94-93a9-e7b232ef17da" name="Default" comment="">
       <change beforePath="$PROJECT_DIR$/.idea/workspace.xml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/workspace.xml" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/out/production/Team5D/sample/Booking.class" beforeDir="false" afterPath="$PROJECT_DIR$/out/production/Team5D/sample/Booking.class" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/out/production/Team5D/sample/TourBookingController.class" beforeDir="false" afterPath="$PROJECT_DIR$/out/production/Team5D/sample/TourBookingController.class" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/out/production/Team5D/sample/TourDataFactory.class" beforeDir="false" afterPath="$PROJECT_DIR$/out/production/Team5D/sample/TourDataFactory.class" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/out/production/Team5D/sample/User.class" beforeDir="false" afterPath="$PROJECT_DIR$/out/production/Team5D/sample/User.class" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/out/production/Team5D/sample/bookingInformation.fxml" beforeDir="false" afterPath="$PROJECT_DIR$/out/production/Team5D/sample/bookingInformation.fxml" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/out/production/Team5D/sample/search.fxml" beforeDir="false" afterPath="$PROJECT_DIR$/out/production/Team5D/sample/search.fxml" afterDir="false" />
       <change beforePath="$PROJECT_DIR$/src/sample/search.fxml" beforeDir="false" afterPath="$PROJECT_DIR$/src/sample/search.fxml" afterDir="false" />
     </list>
     <option name="SHOW_DIALOG" value="false" />
@@ -172,6 +178,17 @@
       <option name="ALTERNATIVE_JRE_PATH_ENABLED" value="false" />
       <option name="ALTERNATIVE_JRE_PATH" />
       <method />
+    </configuration>
+    <configuration default="true" type="ArquillianJUnit" factoryName="" nameIsGenerated="true">
+      <option name="arquillianRunConfiguration">
+        <value>
+          <option name="containerStateName" value="" />
+        </value>
+      </option>
+      <option name="TEST_OBJECT" value="class" />
+      <method v="2">
+        <option name="Make" enabled="true" />
+      </method>
     </configuration>
     <configuration default="true" type="DartUnitRunConfigurationType" factoryName="DartUnit">
       <option name="VMOptions" />
@@ -203,17 +220,6 @@
     </configuration>
     <configuration default="true" type="Application" factoryName="Application">
       <option name="WORKING_DIRECTORY" value="$PROJECT_DIR$" />
-      <method v="2">
-        <option name="Make" enabled="true" />
-      </method>
-    </configuration>
-    <configuration default="true" type="ArquillianJUnit" factoryName="" nameIsGenerated="true">
-      <option name="arquillianRunConfiguration">
-        <value>
-          <option name="containerStateName" value="" />
-        </value>
-      </option>
-      <option name="TEST_OBJECT" value="class" />
       <method v="2">
         <option name="Make" enabled="true" />
       </method>
Index: out/production/Team5D/sample/search.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/out/production/Team5D/sample/search.fxml b/out/production/Team5D/sample/search.fxml
--- a/out/production/Team5D/sample/search.fxml	(revision 8d1c89b7510e1a7c8ba54b9b078d748d9a9e322f)
+++ b/out/production/Team5D/sample/search.fxml	(date 1616168276000)
@@ -15,7 +15,7 @@
    <left>
       <AnchorPane prefHeight="395.0" prefWidth="144.0" BorderPane.alignment="CENTER">
          <children>
-            <SplitMenuButton layoutX="17.0" layoutY="66.0" mnemonicParsing="false" prefWidth="120.0" text="Region">
+            <SplitMenuButton layoutX="17.0" layoutY="88.0" mnemonicParsing="false" prefWidth="120.0" text="Region">
               <items>
                   <CheckMenuItem fx:id="filterNorthRegion" mnemonicParsing="false" text="North" />
                   <CheckMenuItem fx:id="filterEastRegion" mnemonicParsing="false" text="East" />
@@ -26,24 +26,12 @@
                   <Font size="14.0" />
                </font>
             </SplitMenuButton>
-            <SplitMenuButton layoutX="17.0" layoutY="113.0" mnemonicParsing="false" prefWidth="120.0" text="Duration">
+            <SplitMenuButton layoutX="18.0" layoutY="149.0" mnemonicParsing="false" prefWidth="120.0" text="Duration">
               <items>
                   <CheckMenuItem fx:id="filterToThreeHours" mnemonicParsing="false" text="0-3 hours" />
                   <CheckMenuItem fx:id="filterThreeToFiveHours" mnemonicParsing="false" text="3-5 hours" />
                   <CheckMenuItem fx:id="filterFiveToSevenHours" mnemonicParsing="false" text="5-7 hours" />
                   <CheckMenuItem fx:id="filterFullDay" mnemonicParsing="false" text="Full day (7+ hours)" />
-              </items>
-               <font>
-                  <Font size="14.0" />
-               </font>
-            </SplitMenuButton>
-            <SplitMenuButton layoutX="17.0" layoutY="161.0" mnemonicParsing="false" prefWidth="120.0" text="Interests">
-              <items>
-                  <CheckMenuItem fx:id="filterInterestsAdrenaline" mnemonicParsing="false" text="Adrenaline" />
-                  <CheckMenuItem fx:id="filterInterestsRelaxation" mnemonicParsing="false" text="Relaxation" />
-                  <CheckMenuItem fx:id="filterInterestsWaterActivitives" mnemonicParsing="false" text="Water activitives" />
-                  <CheckMenuItem fx:id="filterInterestsNorthernLights" mnemonicParsing="false" text="Northern lights" />
-                  <CheckMenuItem fx:id="filterInterestsHiking" mnemonicParsing="false" text="Hiking" />
               </items>
                <font>
                   <Font size="14.0" />
@@ -65,7 +53,7 @@
          <children>
             <DatePicker fx:id="dateFrom" layoutX="14.0" layoutY="90.0" prefHeight="27.0" prefWidth="120.0" promptText="from" />
             <DatePicker fx:id="dateTo" layoutX="154.0" layoutY="90.0" prefHeight="27.0" prefWidth="120.0" promptText="to" />
-            <Button fx:id="buttonCheckAvailabilty" layoutX="290.0" layoutY="90.0" mnemonicParsing="false" prefHeight="27.0" prefWidth="127.0" text="Check availabilty" />
+            <Button fx:id="buttonFindTour" layoutX="290.0" layoutY="90.0" mnemonicParsing="false" prefHeight="27.0" prefWidth="127.0" text="Find tour" />
             <Text layoutX="14.0" layoutY="54.0" strokeType="OUTSIDE" strokeWidth="0.0" text=" ICELAND" wrappingWidth="164.00634765625">
                <font>
                   <Font size="36.0" />
@@ -73,12 +61,15 @@
             </Text>
             <AnchorPane layoutX="23.0" layoutY="131.0" prefHeight="169.0" prefWidth="249.0">
                <children>
-                  <ListView fx:id="tourListView" layoutX="-6.0" layoutY="1.0" prefHeight="178.0" prefWidth="402.0" />
+                  <ListView fx:id="tourListView" layoutX="-6.0" layoutY="1.0" prefHeight="164.0" prefWidth="402.0" />
                </children>
             </AnchorPane>
             <Text layoutX="108.0" layoutY="74.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Day tours" />
-            <Button fx:id="buttonSearchBookTour" layoutX="274.0" layoutY="328.0" mnemonicParsing="false" prefHeight="27.0" prefWidth="144.0" text="Next step to book tour" />
-            <Button layoutX="329.0" layoutY="14.0" mnemonicParsing="false" text="administrator" />
+            <Button fx:id="buttonSearchBookTour" layoutX="274.0" layoutY="306.0" mnemonicParsing="false" prefHeight="27.0" prefWidth="144.0" text="Next step to book tour" />
+            <Button fx:id="buttonAdministrator" layoutX="329.0" layoutY="14.0" mnemonicParsing="false" text="Administrator" />
+            <Text layoutX="17.0" layoutY="324.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Cancel your booking:" />
+            <TextField fx:id="cancelBookingID" layoutX="16.0" layoutY="334.0" prefHeight="27.0" prefWidth="135.0" promptText="Enter your booking ID" />
+            <Button fx:id="buttonCancel" layoutX="160.0" layoutY="334.0" mnemonicParsing="false" prefHeight="27.0" prefWidth="61.0" text="Cancel" />
          </children>
       </AnchorPane>
    </center>
Index: out/production/Team5D/sample/bookingInformation.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/out/production/Team5D/sample/bookingInformation.fxml b/out/production/Team5D/sample/bookingInformation.fxml
--- a/out/production/Team5D/sample/bookingInformation.fxml	(revision 8d1c89b7510e1a7c8ba54b9b078d748d9a9e322f)
+++ b/out/production/Team5D/sample/bookingInformation.fxml	(date 1616168081000)
@@ -26,20 +26,26 @@
    <right>
       <AnchorPane prefHeight="377.0" prefWidth="403.0" BorderPane.alignment="CENTER">
          <children>
-            <Text layoutX="55.0" layoutY="83.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Name" />
-            <Text layoutX="51.0" layoutY="132.0" strokeType="OUTSIDE" strokeWidth="0.0" text="SSN" />
-            <Text layoutX="53.0" layoutY="179.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Email" />
-            <TextField layoutX="102.0" layoutY="65.0" prefHeight="27.0" prefWidth="268.0" />
-            <TextField layoutX="103.0" layoutY="114.0" prefHeight="27.0" prefWidth="266.0" />
-            <TextField layoutX="103.0" layoutY="161.0" prefHeight="27.0" prefWidth="268.0" />
-            <Button layoutX="216.0" layoutY="257.0" mnemonicParsing="false" prefHeight="27.0" prefWidth="155.0" text="Book tour" />
+            <Text layoutX="54.0" layoutY="105.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Name" />
+            <Text layoutX="58.0" layoutY="154.0" strokeType="OUTSIDE" strokeWidth="0.0" text="SSN" />
+            <Text layoutX="56.0" layoutY="201.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Email" />
+            <TextField layoutX="102.0" layoutY="87.0" prefHeight="27.0" prefWidth="268.0" />
+            <TextField layoutX="103.0" layoutY="136.0" prefHeight="27.0" prefWidth="266.0" />
+            <TextField layoutX="104.0" layoutY="183.0" prefHeight="27.0" prefWidth="268.0" />
+            <Button fx:id="bookTourButton" layoutX="221.0" layoutY="277.0" mnemonicParsing="false" onAction="#bookTourButtonOnAction" prefHeight="27.0" prefWidth="155.0" text="Book tour" />
             <Text layoutX="51.0" layoutY="48.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Booking informations" wrappingWidth="182.21630859375">
                <font>
                   <Font size="18.0" />
                </font>
             </Text>
-            <ChoiceBox layoutX="103.0" layoutY="207.0" prefHeight="27.0" prefWidth="269.0" />
-            <Text layoutX="51.0" layoutY="225.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Persons" />
+            <ChoiceBox layoutX="104.0" layoutY="229.0" prefHeight="27.0" prefWidth="269.0" />
+            <Text layoutX="53.0" layoutY="247.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Persons" />
+            <Text fx:id="showNameOnTour" layoutX="53.0" layoutY="75.0" strokeType="OUTSIDE" strokeWidth="0.0" text="HÉR KEMUR NAFIÐ Á FERÐINNI SEM ER VERIÐ AÐ BÓKA">
+               <font>
+                  <Font size="11.0" />
+               </font>
+            </Text>
+            <Text layoutX="98.0" layoutY="335.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Þegar búið er að ýta á bóka-&gt; bókun tókst bókunar númerið þitt er:" wrappingWidth="245.21630859375" />
          </children>
       </AnchorPane>
    </right>
